### PR TITLE
Crowdplatform code cleanup and fix 

### PR DESCRIPTION
**Pull request by simonkorz**
@fix
errors when connecting to pybossa instance, cause: passing wrong type

### CODE REVIEW

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java [hunk 1]
```diff
@@ -79,12 +79,15 @@ else if (config.database.maintainInterval < 0)
...
                     break;
                 case "pybossa":
-                    platformInstance = new PyBossaPlatform(config.deployment.workerService,
+                    PyBossaPlatform pyBossa = new PyBossaPlatform(config.deployment.workerService,
                             platform.apiKey,
                             platform.url,
                             platform.name,
                             platform.projectId,
                             platform.calibrationsAllowed);
+                    pyBossa.init();
```
Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:88]: why having a seperated init function which is called all the time after the constructor?

**Comment by marcelhollerbach** [comment]
please update

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java (hunk 1 above)
Comment (simonkorz) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:88]: it initializes the platform and needs a connection to it. I think you should still be able to construct the platform then

Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:88]: mhmmm but what is the purpose of it if you have a object in a unusable state if you dont call init before?

Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:88]: Being able to test parts of it without internet.

Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java:88]: how do you want to test pybossa without internet ? :-D

**Pull request merged**